### PR TITLE
Autocomplete function Done

### PR DESCRIPTION
**Pull request by JeonHyungJin**
I added autocomplete function.
when you type some email in inputbox, you can see the autocompleted email lists under the inputbox.
This can save your time to type those things.

If you want to add other domains you can fix your code.
https://github.com/JeonHyungJin/react-multi-email/blob/master/src/react-multi-email/ReactMultiEmail.tsx#L70

```
result = ['gmail.com', 'naver.com', 'daum.net'].map(domain => `${value}@${domain}`);
```
If you want to add, modify the domain list, try to modify this result list!

### CODE REVIEW

**Comment by JeonHyungJin** [comment]
I changed isEmail, too.

File: src/react-multi-email/ReactMultiEmail.tsx [hunk 1]
```diff
@@ -61,7 +62,14 @@ class ReactMultiEmail extends React.Component<IReactMultiEmailProps> {
       return true;
     };
 
+    let result;
     if (value !== '') {
+      if (!value || value.indexOf('@') >= 0) {
+        result=[]
+      } else {
+        result = ['gmail.com', 'naver.com', 'daum.net'].map(domain => `${value}@${domain}`);
```
[the remark this answers is not in the document]
Comment (thomasJang) [reply src/react-multi-email/ReactMultiEmail.tsx]: And, suffix autocompletion should only work if suffixDomains is present.

File: src/react-multi-email/style.css [hunk 2]
```diff
@@ -74,3 +74,27 @@
   margin-left: 0.833em;
   cursor: pointer;
 }
+ul{
+  list-style: none;
+  padding: 0;
+}
+.result_list.empty {
```
Comment (thomasJang) [inline src/react-multi-email/style.css]: When `result-list` appears, it should appear as a pop-up.
image
like this image



You can find more detail action here.
https://ant.design/components/auto-complete/

Comment (thomasJang) [reply src/react-multi-email/style.css]: You can do it

File: src/react-multi-email/ReactMultiEmail.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (thomasJang) [reply src/react-multi-email/ReactMultiEmail.tsx]: You can use `this.props.suffixDomains` at findEmailAddress function.

```js
 findEmailAddress = (value: string, isEnter?: boolean) => {
    const {suffixDomains = []} = this.props;

    if(suffixDomains.length){
      suffixDomains.map(....
    }
  };
```

**Pull request merged**